stps/sig-network/ip-request-stp.md

```diff
@@ -0,0 +1,216 @@
+| **Technology Challenges**         | [V]   | Identified potential testing challenges related to the underlying technology.                                                                            |                                                                                                                                                     |
+| **Test Environment Needs**        | [V]   | Determined necessary **test environment setups and tools**.                                                                                              |                                                                                                                                                     |
+| **API Extensions**                | [V]   | Reviewed new or modified APIs and their impact on testing.                                                                                               |                                                                                                                                                     |
+| **Topology Considerations**       | [V]   | Evaluated multi-cluster, network topology, and architectural impacts.                                                                                    | Same as in the setups that are used for the MTV IPAM claim testing.<br/>3rd party will provide UDN and annotated VM, CNV testing starts from that.  |
+
+
+### **II. Software Test Plan (STP)**
+
+This STP serves as the **overall roadmap for testing**, detailing the scope, approach, resources, and schedule.
+
+#### **1. Scope of Testing**
+
+Briefly describe what will be tested. The scope must **cover functional and non-functional requirements**. Must ensure user stories are included and aligned to downstream user stories from Section I.
+
+**In Scope:**
+- Testing of known networking flows on VMs, where specific IP is requested for primary interface over
+Cluster/UserDefinedNetwork.\
+The uniqueness in this feature is how the IP is assigned - using the new `network.kubevirt.io/addresses` annotation
+
+**Document Conventions (if applicable):** None
+
+#### **2. Testing Goals**
+
+Define specific, measurable testing objectives for this feature, such as:
+
+- [ ] Goal 1: Verify known networking scenarios are still valid when primary static IP is assigned via the new\
+annotation.
+- [ ] [Goal 2: Consider new cases to be tested, presented by this new feature.
+
+#### **3. Non-Goals (Testing Scope Exclusions)**
+
+Explicitly document what is **out of scope** for testing. **Critical:** All non-goals require explicit stakeholder agreement to prevent "I assumed you were testing that" issues.
```

Comment:
what about:
- IPv6 static IP allocation scenarios
- secondary interface
- IP addresses must be within the subnet range